src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.ts

```diff
@@ -541,6 +544,15 @@ export class TableComponent implements AfterContentChecked, OnInit, OnChanges, O
           } else if (_.isNumber(cellValue) || _.isBoolean(cellValue)) {
             cellValue = cellValue.toString();
           }
+
+          if (_.isObjectLike(cellValue)) {
+            if (this.searchableObjects) {
+              cellValue = JSON.stringify(cellValue);
```

Comment:
Nice solution to fix this.

Reply:
Nice fix. One thing might confuse users is data in objects are searchable, but they are not displayed after applying the cell template. But I don't think we have this case.

Reply:
As the default case is, to not search an object, we should be fine here.